src/protocol/templates/ping_doc.in

```diff
@@ -6,9 +6,9 @@
 * [Format](#format)
 {% for sensor in sensors | sort %}
 {% set messages = sensors[sensor]['messages'] %}
-* [{{sensor}} Messages](#{{sensor}}-messages)
+* [{{generator.capitalize(sensor)}} Messages](#{{generator.capitalize(sensor)}}-messages)
```

Comment:
you can just use the capitalize filter: ` {{ sensor|capitalize }}`

Reply:
Our capitalize function moves from snake case to camel case, it's not a simple capitalize one.